Packs/CommonTypes/Layouts/layoutscontainer-Vulnerability.json

```diff
@@ -4,6 +4,7 @@
     "name": "Vulnerability Incident",
     "version": -1,
     "fromVersion": "6.0.0",
+    "toVersion": "6.9.9",
```

Comment:
As it's not indicator layout (but incident) let's revert this layout change

Reply:
Yep, that happens when manually editing so many files :)